platforms/r3-corda-ent/charts/cenm-gateway/values.yaml

```diff
@@ -4,217 +4,54 @@
-  # this chart doesn't create the storageclass.
-  # Eg. name: cordaentsc
-  name: cordaentsc
-
-# Required parameter to start any .jar files
-# e.g. acceptLicense: YES 
-acceptLicense: YES
-
-#############################################################
-#               HashiCorp Vault Configuration               #
-#############################################################
-# NOTE: Make sure that the vault is already unsealed, intialized and configured to 
-# use Kubernetes service account token based authentication. 
-# For more info, see https://www.vaultproject.io/docs/auth/kubernetes
-
-vault:
-  # Provide the vault address
-  # Eg. address: http://vault.example.com:8200
-  address: 
-  # Provide the vault role used.
-  # Eg. role: vault-role
-  role: vault-role
-  # Provide the authpath configured to be used.
-  # Eg. authpath: entcordacenm
-  authPath: entcordacenm
-  # Provide the service account name autheticated to vault.
-  # NOTE: Make sure that the service account is already created and autheticated to use the vault.
-  # Eg. serviceAccountName: vault-auth
+global:
   serviceAccountName: vault-auth
-  # Provide the vault path where the  certificates are stored
-  # Eg. certsecretprefix: secret/cenm-org-name/signer/certs
-  certSecretPrefix: 
-  # The amount of times to retry fetching from/writing to Vault before giving up.
-  # Eg. retries: 10
-  retries: 10
- # Specify the maximum size of the memory allocation pool
-
-#############################################################
-#                       Settings                            #
-#############################################################
-config:
-  # Provide volume related specifications
-  volume:
-    baseDir:
-
-  # Provide the path where the CENM Idman .jar-file is stored
-  # Eg. jarPath: bin
-  jarPath: bin
-
-  # Provide the path where the CENM Service configuration files are stored
-  # Eg. configPath: etc
-  configPath: 
-
-  # Provide any extra annotations for the PVCs
-  pvc:
-    #   annotations:
-    #     key: "value"
-    annotations: {}
-    # Volume size for etc/ directory
-    volumeSizeGatewayEtc: 1Gi
-    # Volume size for logs/ directory
-    volumeSizeGatewayLogs: 5Gi
+  cluster:
+    provider: aws  # choose from: minikube | aws | azure | gcp
+    cloudNativeServices: false # set to true to use Cloud Native Services (SecretsManager and IAM for AWS; KeyVault & Managed Identities for Azure)
+  vault:
+    type: hashicorp
+    role: vault-role
+    address: 
+    authPath: supplychain
+    secretEngine: secretsv2
+    secretPrefix: "data/supplychain"
+  proxy:
+    #This will be the proxy/ingress provider. Can have values "ambassador" or "none"
+    #Eg. provider: "ambassador"
+    provider: "ambassador"
+    #This field contains the external URL of the node
+    #Eg. externalUrlSuffix: test.blockchaincloudpoc.com
+    externalUrlSuffix: test.blockchaincloudpoc.com
+  cenm:
```

Comment:
What is the idea behind CENM details in global? Are these keys getting repeated in unrelated Charts?

Reply:
yeah, for example the sharedCreds and gateway ports are used in almost all charts